addon/src/main/java/com/jaimemartz/playerbalanceraddon/MainCommand.java

```diff
@@ -14,6 +14,11 @@ public MainCommand(PlayerBalancerAddon plugin) {
         this.plugin = plugin;
     }
 
+    // Command: /spb 
+    // <connect | fallback | bypassconnect | setbypass | clearbypass | overridestatus | clearoverride>
+    // <section | server | player> [player]
+    // example: /spb connect <section> [player]
+    // or: /sbp bypassconnect <server> [player]
```

Comment:
For what are these commands for? Create a README for stuff like this

Reply:
It's always good to have documentation in the files, so that development is easier. 
In this case, I needed an easy way to determine how the command worked.

Reply:
These may be good for yourself, but it's useless for other people. Code should be clear and not filled with some command syntaxes, that's something why READMEs exist.

Reply:
All good code should be well documented, either within or outside of the codebase, at any given time. Since I could not find any documentation for this specific part of the code, I added some myself.

Another good thing to note, I did all of this in the online version of Github's editor. It makes things a little bit harder to work when outside of a proper IDE.

Reply:
Alright :+1: